.travis.yml

```
@@ -46,7 +46,8 @@ jobs:
     # - run tests without pinned dependencies to catch issues with new versions
 
     - stage: additional tests
-      env: NAME=test sauce
+      name: test sauce
+      env: SAUCE_VERSION='latest'
```

Apparently I was wrong and `latest` doesn't work :sob:

Can you tweak to:

```suggestion
      env: SAUCE_VERSION='4.5.4'
```

(afaict that is the latest version from their [changelog](https://wiki.saucelabs.com/display/DOCS/Sauce+Connect+Proxy+Change+Logs))

---

Running now.